VisualPinball.Unity/VisualPinball.Unity.Editor/Managers/SoundManager.cs

```diff
@@ -109,15 +111,59 @@ protected override void OnEnable()
 				fixedHeight = 25.0f,
 				fixedWidth = 25.0f
 			};
+
 			base.OnEnable();
 			SceneView.duringSceneGui += OnSceneGUI;
 		}
 
 		protected void OnDisable()
 		{
+			GameObject.DestroyImmediate(_audioSource);
+			_audioSource = null;
+			_audioSourceComp = null;
 			SceneView.duringSceneGui -= OnSceneGUI;
 		}
 
+		public static string[] _soundOutTypeStrings = {
+			"Table",
+			"Backglass",
+		};
+		private static byte[] _soundOutTypeValues = {
+			SoundOutTypes.Table,
+			SoundOutTypes.Backglass,
+		};

```

Comment:
Maybe move these more to the top of the file? And would `_soundOutTypeStrings` be private too?